### PR TITLE
New version: GarboMuffin.TurboWarp version 1.7.1 [FP-O]

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | TurboWarp |     |
| Version     | 1.7.1     |  |
| Publisher   | Thomas Weber   |       |
| ProductCode |           |     |
#### Auto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation) in workflow run [5527](<https://github.com/vedantmgoyal2009/vedantmgoyal2009/actions/runs/4003618365>)

### CODE REVIEW

**Comment by stephengillie** [comment]
From [Turbowarp Release v1.7.0](https://github.com/TurboWarp/desktop/releases/tag/v1.7.0): 
> This version has been yanked and superceded by 1.7.0

File: manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-8
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: GarboMuffin.TurboWarp
+PackageVersion: 1.7.0
```
Comment (stephengillie) [inline manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.installer.yaml:5]:
```suggestion
PackageVersion: 1.7.1
```

File: manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/TurboWarp/desktop/releases/download/v1.7.0/TurboWarp-Setup-1.7.0-x64.exe
+  InstallerSha256: CC3EF5CAACF34BE700D9423B9D6392FB85D48AF143BC4AD170314028EC78BC42
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/TurboWarp/desktop/releases/download/v1.7.0/TurboWarp-Setup-1.7.0-x64.exe
+  InstallerSha256: CC3EF5CAACF34BE700D9423B9D6392FB85D48AF143BC4AD170314028EC78BC42
```
Comment (stephengillie) [inline manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.installer.yaml:41]:
```suggestion
  InstallerUrl: https://github.com/TurboWarp/desktop/releases/download/v1.7.1/TurboWarp-Setup-1.7.1-x64.exe
  InstallerSha256: 681805552D15218835044060070ED58FE9F7121D8FCDAD986A7520A999F7123C
```

File: manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/TurboWarp/desktop/releases/download/v1.7.0/TurboWarp-Setup-1.7.0-ia32.exe
+  InstallerSha256: E5ED0AF72F7B68B1B055B2513325281371C72B8C141983F0DC0CC86C9D505555
+  InstallerSwitches:
+    Custom: /ALLUSERS
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/TurboWarp/desktop/releases/download/v1.7.0/TurboWarp-Setup-1.7.0-x64.exe
+  InstallerSha256: CC3EF5CAACF34BE700D9423B9D6392FB85D48AF143BC4AD170314028EC78BC42
```
Comment (stephengillie) [inline manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.installer.yaml:35]:
```suggestion
  InstallerUrl: https://github.com/TurboWarp/desktop/releases/download/v1.7.1/TurboWarp-Setup-1.7.1-x64.exe
  InstallerSha256: 681805552D15218835044060070ED58FE9F7121D8FCDAD986A7520A999F7123C
```

File: manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+FileExtensions:
+- sb
+- sb2
+- sb3
+ReleaseDate: 2023-01-25
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/TurboWarp/desktop/releases/download/v1.7.0/TurboWarp-Setup-1.7.0-ia32.exe
+  InstallerSha256: E5ED0AF72F7B68B1B055B2513325281371C72B8C141983F0DC0CC86C9D505555
```
Comment (stephengillie) [inline manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.installer.yaml:23]:
```suggestion
  InstallerUrl: https://github.com/TurboWarp/desktop/releases/download/v1.7.1/TurboWarp-Setup-1.7.1-ia32.exe
  InstallerSha256: A881AC6773F97CA18F97CECD6E2B1C6CCCF1DDCC96BB4A43E253601D2901D70B
```

File: manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/TurboWarp/desktop/releases/download/v1.7.0/TurboWarp-Setup-1.7.0-ia32.exe
+  InstallerSha256: E5ED0AF72F7B68B1B055B2513325281371C72B8C141983F0DC0CC86C9D505555
```
Comment (stephengillie) [inline manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.installer.yaml:29]:
```suggestion
  InstallerUrl: https://github.com/TurboWarp/desktop/releases/download/v1.7.1/TurboWarp-Setup-1.7.1-ia32.exe
  InstallerSha256: A881AC6773F97CA18F97CECD6E2B1C6CCCF1DDCC96BB4A43E253601D2901D70B
```

File: manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-8
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: GarboMuffin.TurboWarp
+PackageVersion: 1.7.0
```
Comment (stephengillie) [inline manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.7.1
```

File: manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+ShortDescription: TurboWarp is a mod of Scratch with a compiler and more features.
+# Description:
+# Moniker:
+Tags:
+- electron
+- scratch
+- wrapper
+# Agreements:
+# ReleaseNotes:
+ReleaseNotesUrl: https://github.com/TurboWarp/desktop/releases/tag/v1.7.0
```
Comment (stephengillie) [inline manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.locale.en-US.yaml:27]:
```suggestion
ReleaseNotesUrl: https://github.com/TurboWarp/desktop/releases/tag/v1.7.1
```

File: manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.yaml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-8
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: GarboMuffin.TurboWarp
+PackageVersion: 1.7.0
```
Comment (stephengillie) [inline manifests/g/GarboMuffin/TurboWarp/1.7.0/GarboMuffin.TurboWarp.yaml:5]:
```suggestion
PackageVersion: 1.7.1
```

**Comment by stephengillie** [comment]
A path update is also needed.

**Review by ImJoakim — APPROVED** [review]

**Pull request merged**